### PR TITLE
Genegram executable

### PR DESCRIPTION
**Pull request by vadyushkins**
Created a simple executable (`python -m genegram`) from a package (`genegram`). Accepts a file path to seq.fasta (`--seq_fasta` argument) and folder path to save the results to (`--predict` argument).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+## Usage
+
+Run the `python -m generam` with the arguments.
+
+### **Required arguments**
+
+Argument | Description
+:--- | :---
+--seq_fasta | Path to the `seq.fasta` file
+--predict | Path to the folder where the predictions will be saved
```
Comment (gsvgit) [inline README.md]: ```predict``` looks like a task (predict, filter, rebase, etc), not a path to the folder. Maybe, we can use more typical ```--out``` or something similar.

File: genegram/__main__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import tempfile
+from argparse import ArgumentParser
+from pathlib import Path
+
+from genegram import seq_fasta_to_pictures, predict
+
+if __name__ == "__main__":
+    parser = ArgumentParser(description="Genegram")
+    parser.add_argument("--seq_fasta", required=True, type=str, help="input data path")
```
Comment (gsvgit) [inline genegram/__main__.py]: A bit more information on input format is required.

File: genegram/__main__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    parser.add_argument(
+        "--predict",
+        required=True,
+        type=str,
+        help="output data path",
```
Comment (gsvgit) [inline genegram/__main__.py]: More details: it is a folder, etc.

File: genegram/model.py [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+# Model definition functions
+
+# layer that for inputs i1, i2, i3, i4
+# returns (w1*i1 + w2*i2 + w3*i3 + w4*i4) / 4, where wi are trainable coeffs
+class WeightedSum(Layer):
+    def __init__(self):
+        super(WeightedSum, self).__init__()
+        w_init = tf.keras.initializers.Ones()
+        self.w1 = tf.Variable(
+            initial_value=w_init(shape=(), dtype="float32"), trainable=True
```
Comment (gsvgit) [inline genegram/model.py]: Why layers are trainable? I guess, model training and model using should be separated.

Comment (vivalerievich) [reply genegram/model.py]: Unfortunately, I don't know how to do it better yet. @LuninaPolina, can you help please? :eyes:

**Comment by vadyushkins** [comment]
I think the current PR is ready for merge. @gsvgit, @LuninaPolina what do you say?

**Review by gsvgit — APPROVED** [review]

**Pull request merged**